src/coq_elpi_builtins_synterp.ml

```diff
@@ -809,31 +829,34 @@ module SynterpAction = struct
 
     MLCode(Pred("coq.begin-synterp-group",
       In(id, "ID",
-      Full(unit_ctx,"Match a the begin-synterp-group synterp operation. ID must be the next synterp action group and there must not be any actions before it.")),
-    (fun name ~depth _ _ state ->
-      State.update read state (fun rz ->
-        match ReadZipper.open_group rz name with
-        | Ok rz -> rz
-        | Error `Empty ->
-          synterp_interp_error Pp.(hov 0 (strbrk ("The command tried to match a begin-synterp-group operation on group " ^ name ^ " but there are no groups or actions left in the current state.")))
-        | Error (`GroupMismatch name') ->
-          synterp_interp_error Pp.(hov 0 (strbrk ("The command tried to match a begin-synterp-group operation on group " ^ name ^ " but the next group to match has name " ^ name' ^ ".")))
-        | Error `LeadingAction ->
-          synterp_interp_error Pp.(hov 0 (strbrk ("The command tried to match a begin-synterp-group operation on group " ^ name ^ " but the next item to match is an action, not a group.")))
-      ), (), [])),
+      Out(Tree.group, "Group",
```

Comment:
You also want to declare a `MLData Tree.group` so that it shows up in the generated file.

Reply:
Done.